### PR TITLE
Reintroduce artificial delay on block generator

### PR DESCRIPTION
**Pull request by herr-seppia**

### CODE REVIEW

File: pkg/config/groups.go [hunk 1]
```diff
@@ -137,6 +137,8 @@ type mempoolConfiguration struct {
 
 	// Enables mempool updates at startup
 	Updates updates
+
+	ExtractionDelaySecs uint32
```
Comment (goshawk-3) [inline pkg/config/groups.go:141]: nitpick. Add comment here to explain the new field.

**Review by goshawk-3 — CHANGES_REQUESTED** [review]
Maybe it's also good to benchmark the PR with empty and full blocks in devnet to ensure we have a predictable result.

**Review by goshawk-3 — APPROVED** [review]

**Comment by herr-seppia** [comment]
> @goshawk-3 
> Maybe it's also good to benchmark the PR with empty and full blocks in devnet to ensure we have a predictable result.

Prior to this PR:

image
image
image
image

With this PR
image
image
image
image

**Comment by goshawk-3** [comment]
Looks very good @herr-seppia

**Pull request merged**